pkg/gadgets/trace/capabilities/tracer/tracer.go

```diff
@@ -282,16 +302,20 @@ func (t *Tracer) run() {
 				Type:      eventtypes.NORMAL,
 				Timestamp: gadgets.WallTimeFromBootTime(bpfEvent.Timestamp),
 			},
-			MountNsID: bpfEvent.Mntnsid,
-			Pid:       bpfEvent.Pid,
-			Cap:       int(bpfEvent.Cap),
-			UID:       bpfEvent.Uid,
-			Audit:     audit,
-			InsetID:   insetID,
-			Comm:      gadgets.FromCString(bpfEvent.Task[:]),
-			Syscall:   syscall,
-			CapName:   capabilityName,
-			Verdict:   verdict,
+			MountNsID:     bpfEvent.Mntnsid,
+			TargetUserNs:  bpfEvent.TargetUserns,
+			CurrentUserNs: bpfEvent.CurrentUserns,
+			Pid:           bpfEvent.Pid,
+			Cap:           int(bpfEvent.Cap),
+			UID:           bpfEvent.Uid,
+			Audit:         audit,
+			InsetID:       insetID,
+			Comm:          gadgets.FromCString(bpfEvent.Task[:]),
+			Syscall:       syscall,
+			CapName:       capabilityName,
+			Verdict:       verdict,
+			Caps:          bpfEvent.CapEffective,
+			CapsNames:     capsNames(bpfEvent.CapEffective),
```

Comment:
I think it's interesting to send the full list of capabilities so the caller doesn't have to parse them, but, at the same time I'm a bit confused about performance, it seems we're sending too much data for each event and some clients could parse it to reduce the network usage. 

Perhaps this could  be configurable, on by default, and kubectl-gadget could disable it. @flyth do you have any opinions on this kind of situation?

btw, I'm completely fine if we handle this later on.

Reply:
I also worry a bit about these extremely long strings. Making it configurable sounds good - we can easily do that with the upcoming kubectl upgrade PR and have it convert to strings on the client by default. So I'd propose to leave it as it is for now and optimize once the other PR is merged.

Reply:
Ok, I will keep it as is.

I am not sure it is worth optimising for this specific gadget: the stream of capability events are not high throughput like it would be for a tcpdump gadget.